### PR TITLE
add info panel

### PR DESCRIPTION
**Pull request by domgan**
* info panel with tutorial on basic controls and roadmap
* basic css, we can expand on background style etc
* improved aside section
* created `buttons.scss`
image

### CODE REVIEW

File: src/common/styles/buttons.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    cursor: pointer;
+    z-index: 100;
+    border: 2px solid $color-primary;
+    border-radius: 0.7rem;
+
+    &:hover {
+        background-color: $color-primary;
+        transform: translateY(-1px);
+    }
+}
```
Comment (matfij) [inline src/common/styles/buttons.scss:17]: we have some other common button classes in `utils.scss` should probably be moved here, also other button-related classes were abbreviated `btn` so we should either update them to `button` or rename this new one to `btn`.

File: src/views/game/GameComponent.tsx [hunk 2]
```diff
@@ -66,6 +67,18 @@ export const GameComponent = () => {
         musicService.stopCurrent();
     };
 
+    const AsidePanel = () => (
```
Comment (matfij) [inline src/views/game/GameComponent.tsx:70]: why mixing rendering with other if no arguments are required?

**Review by matfij — CHANGES_REQUESTED** [review]
Instead of a roadmap (which should be included in some external resource (like public notion page/wiki or github pages) we should add instructions for battles.

File: src/views/game/GameComponent.tsx (hunk 2 above)
Comment (domgan) [reply src/views/game/GameComponent.tsx:70]: Dunno what you mean

**Comment by domgan** [comment]
* I think roadmap is a nice thing for players to see so they know what to expect and what was done
* Agreed with battle instructions. I myself don't know how it works so feel free to expand yourself

**Comment by domgan** [comment]
* I like having a guide post as a general place of comms with players, at least for now. We can expand on that idea.
* I think we should explain basics of resistances, physical attacks and defenses.

File: src/views/game/GameComponent.tsx (hunk 2 above)
Comment (matfij) [reply src/views/game/GameComponent.tsx:70]: Why create a function for rendering it when no arguments are required to render and it makes component more convoluted by placing TSX where usually TS is placed.

Comment (domgan) [reply src/views/game/GameComponent.tsx:70]: > Why create a function for rendering it when no arguments are required to render

And why create functions? Code organization and readability.

> it makes component more convoluted by placing TSX where usually TS is placed

Why "usually"? That's not aligned with my experience.

**Review by matfij — APPROVED** [review]
LGTM

**Pull request merged**